### PR TITLE
Feature/mo pub ad bidding

### PR DESCRIPTION
**Pull request by orkhanalizada2020**

### CODE REVIEW

**Review by can-soykarafakili — APPROVED** [review]
Looks good from my end!! How about you @eros902002 ?

File: PubnativeLite/PubnativeLite/Utils/Header Bidding/HyBidHeaderBiddingUtils.m [hunk 1]
```diff
@@ -67,4 +67,27 @@ + (NSString *)eCPMFromAd:(HyBidAd *)ad withDecimalPlaces:(HyBidKeywordMode)decim
     }
 }
 
++ (NSString *)createAppMonetHeaderBiddingKeywordsStringWithAd:(HyBidAd *)ad
```
Comment (eros902002) [inline PubnativeLite/PubnativeLite/Utils/Header Bidding/HyBidHeaderBiddingUtils.m:70]: There's a method missing for interstitial, no?

File: PubnativeLite/PubnativeLiteDemo/Adapters/AppMonet Adapters/MoPub Adapters/AppMonet+MoPub.m [hunk 2]
```diff
@@ -22,15 +22,113 @@
...
+
+@interface AppMonet ()
+typedef struct { MPAdView *adView; void (^onReadyBlock)(void); } AppMonetAdStruct;
+@property (class, nonatomic) NSMutableDictionary *dict;
+@end
 
 @implementation AppMonet (MoPub)
 
-+ (void)addNativeBids:(id)adRequest andAdUnitId:(NSString *)adUnitId andTimeout:(NSNumber *)timeout :(void (^)(void))onReadyBlock {
++ (void)addBids:(MPAdView *)adView andAppMonetAdUnitId:(NSString *)appMonetAdUinitId andTimeout:(NSNumber *)timeout :(void (^)(void))onReadyBlock {
```
Comment (eros902002) [inline PubnativeLite/PubnativeLiteDemo/Adapters/AppMonet Adapters/MoPub Adapters/AppMonet+MoPub.m:34]: there should be one method to put bids on interstitial as well, no?

File: PubnativeLite/PubnativeLite/Utils/Header Bidding/HyBidHeaderBiddingUtils.m (hunk 1 above)
Comment (eros902002) [reply PubnativeLite/PubnativeLite/Utils/Header Bidding/HyBidHeaderBiddingUtils.m:70]: https://github.com/pubnative/pubnative-hybid-android-sdk/blob/feature/OMA-832-hybid-extension/hybid.sdk/src/main/java/net/pubnative/lite/sdk/utils/HeaderBiddingUtils.java#L149

File: PubnativeLite/PubnativeLiteDemo/Adapters/AppMonet Adapters/MoPub Adapters/AppMonet+MoPub.m (hunk 2 above)
Comment (can-soykarafakili) [reply PubnativeLite/PubnativeLiteDemo/Adapters/AppMonet Adapters/MoPub Adapters/AppMonet+MoPub.m:34]: ahh correct 👍

File: PubnativeLite/PubnativeLite/Utils/Header Bidding/HyBidHeaderBiddingUtils.m (hunk 1 above)
Comment (orkhanalizada2020) [reply PubnativeLite/PubnativeLite/Utils/Header Bidding/HyBidHeaderBiddingUtils.m:70]: @eros902002 correct. Yesterday and today I was focused only on the banner side. That's why I skipped here any interstitional method :)
But let me add it 👍🏻

File: PubnativeLite/PubnativeLite/Utils/Header Bidding/HyBidHeaderBiddingUtils.m [hunk 3]
```diff
@@ -90,4 +90,27 @@ + (NSString *)createAppMonetHeaderBiddingKeywordsStringWithAd:(HyBidAd *)ad
...
+{
+    NSNumber *eCPM = (ad.eCPM != nil) ? ad.eCPM : 0;
+    NSMutableString *keywords;
+    NSString *keywordSuffix;
+    
+    if (eCPM.integerValue < 1000) {
+        keywordSuffix = @"mm_10:";
+    } else if (eCPM.integerValue < 2000) {
+        keywordSuffix = @"mm_50:";
+    } else if (eCPM.integerValue < 5000) {
```
Comment (eros902002) [inline PubnativeLite/PubnativeLite/Utils/Header Bidding/HyBidHeaderBiddingUtils.m:103]: Here the limit should be 10000

File: PubnativeLite/PubnativeLite/Utils/Header Bidding/HyBidHeaderBiddingUtils.m [hunk 4]
```diff
@@ -90,4 +90,27 @@ + (NSString *)createAppMonetHeaderBiddingKeywordsStringWithAd:(HyBidAd *)ad
...
+        keywordSuffix = @"mm_1d:";
+    } else {
+        keywordSuffix = @"mm_gte_10d:";
+    }
+    
+    NSString *body = (eCPM.integerValue >= 5000) ? @"true" : [NSString stringWithFormat:@"%.02f", ad.eCPM.doubleValue];
```
Comment (eros902002) [inline PubnativeLite/PubnativeLite/Utils/Header Bidding/HyBidHeaderBiddingUtils.m:109]: Same for this I think

**Review by faresbh — APPROVED** [review]

**Review by eros902002 — APPROVED** [review]
Looks good

**Pull request merged**